src/action.js

```diff
@@ -362,7 +365,7 @@ export function dndzone(node, options) {
             const draggableEl = node.children[idx];
             styleDraggable(draggableEl, dragDisabled);
             if (config.items[idx].hasOwnProperty('isDndShadowItem')) {
-                morphDraggedElementToBeLike(draggedEl, draggableEl, currentMousePosition.x, currentMousePosition.y);
+                morphDraggedElementToBeLike(draggedEl, draggableEl, currentMousePosition.x, currentMousePosition.y, config.transformDraggedElement, draggedElData);
```

Comment:
draggedElData is not really needed inside (it is only passed through).
can we instead do something like:
```javascript
morphDraggedElementToBeLike(draggedEl, draggableEl, currentMousePosition.x, currentMousePosition.y, () => config.transformDraggedElement(draggedEl, draggedElData));
```
?

Reply:
also, maybe it is a good idea to also pass 'idx' (the index the dragged element will become) into transformDraggedElement... what do u think? useful?

Reply:
I haven't necessarily had a use case for the idx, but it doesn't hurt to send more data just in case. Will update.

Reply:
Updated